### PR TITLE
Add an important feature from IRAF

### PR DESCRIPTION
**Pull request by eteq**
To use, just do:
```
import astropy

astropy.flpr()
```
I think I got both the style of the documentation and the actual implementation right from the original version...

### CODE REVIEW

File: astropy/__init__.py [hunk 1]
```diff
@@ -239,3 +239,16 @@ def online_help(query):
         del locals()[varname]
 
 del varname, __module_type__
+
+
+def flpr():
```
Comment (barentsen) [inline astropy/__init__.py:244]: Great work! :+1:

Prayers to the gods of computing occasionally take a long time to be answered.  Can I suggest we make this an asynchronous function?
```suggestion
async def flpr():
```

**Review by barentsen — CHANGES_REQUESTED** [review]

**Review by pllim — COMMENTED** [review]
I think it should randomly throw a kernel panic.

**Comment by pllim** [comment]
And based on https://www.noao.edu/kpno/manuals/ice/node7.html , might need to call it twice, just to be safe.

**Comment by eteq** [comment]
Good points @pllim !  See https://github.com/astropy/astropy/pull/11460/commits/d25b1d14462dc6bcc8b519ca5af5ae7a1cfd7ccb where I added some code inspired by that link.

File: astropy/__init__.py [hunk 2]
```diff
@@ -239,3 +239,20 @@ def online_help(query):
...
+    import random
+    from urllib.error import URLError
+    from urllib.parse import urlencode
+    from urllib.request import Request, urlopen
+
+    if random.random() > .8:
+        print('Abnormal termination of child process. Or something 🤷')
```
Comment (pllim) [inline astropy/__init__.py:251]:
```suggestion
        print('PANIC in `/the/disco.e\': Abnormal termination of child process. Or something 🤷')
```

**Comment by pllim** [comment]
Alright, we got our fun. Until next year! 😉

**Pull request closed without merging**